src/python/traces/sampling/recipefile.json

```
@@ -0,0 +1,31 @@
+{
+  "$schema": "../../../../otel-recipes-schema.json",
+  "id": "python.traces.sampling",
```

```suggestion
  "id": "python.traces.traceidratio.sampler",
```

To match the one I suggested above